CHANGELOG.md

```diff
@@ -1,6 +1,70 @@
 # go-ipfs changelog
 
-## 0.4.16-rc1 2018-06-27
```

Comment:
Header missing?

Reply:
Yeah, i'm really not sure how to format this anymore.